target/linux/ramips/dts/mt7621_xiaomi_mi-router-4a-gigabit-v2.dts

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621_xiaomi_mi-router-4a-common.dtsi"
+
+/ {
+	compatible = "xiaomi,mi-router-4a-gigabit-v2", "mediatek,mt7621-soc";
+	model = "Xiaomi Mi Router 4A Gigabit Edition v2";
+
+	aliases {
+		label-mac-device = &wan;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status_blue: status_blue {
+			label = "blue:status";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+
+		led_status_yellow: status_yellow {
+			label = "yellow:status";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		wan_blue {
+			label = "blue:wan";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		wan_yellow {
+			label = "yellow:wan";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&partitions {
+	partition@180000 {
+		// size changed against to the common dtsi
+		reg = <0x180000 0xe70000>;
+	};
+
+	partition@ff0000 {
+		label = "Config";
+		reg = <0xff0000 0x10000>;
+		read-only;
+	};
+};
+
+&pcie0 {
+	wifi@0,0 {
+		compatible = "pci14c3,7603";
```

Comment:
Why PCI extresses is here? I'd suggest to use `compatible = "mediatek,mt76";`. You can look at the documentation here: https://elixir.bootlin.com/linux/v5.10.163/source/Documentation/devicetree/bindings/net/wireless/mediatek,mt76.txt#

You can use this compatible for all of the mt76xx and move it to dtsi.